bigchaindb/commands/utils.py

```diff
@@ -3,18 +3,39 @@
 """
 
 import argparse
+import builtins
+import functools
 import multiprocessing as mp
 import subprocess
+import sys
 
 import rethinkdb as r
 from pymongo import uri_parser
 
 import bigchaindb
+import bigchaindb.config_utils
 from bigchaindb import backend
 from bigchaindb.common.exceptions import StartupError
 from bigchaindb.version import __version__
 
 
+def configure_bigchaindb(command):
```

Comment:
Just as a side note: perhaps we can find a better name, since `configure_bigchaindb` does sound like it could do just what `autoconfigure` does.